src/pudl/extract/eiaaeo.py

```diff
@@ -311,7 +314,7 @@ def __sanitize(self, s: str) -> str:
         return re.sub(self.__sanitize_re, "_", s.lower().strip().replace(" : ", "__"))
 
     def __series_to_records(
-        self, series_id: str, potential_parents: set[int]
+        self, series_id: str, potential_parents: set[int], year: int
```

Comment:
Slight preference for using `report_year` instead of `year` for this name, if only because I've been tripped up by the report vs model case vs prediction year values before